src/index.html

```diff
@@ -10,7 +10,33 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" method="post" class="form-field">
```

Comment:
there is should be two different forms instead of one
```
<form>
  <label>
    <img>
    <input>
  </label>
</form>

<form>
  <label>
    <img>
    <input>
  </label>
</form>
```